src/player.cpp

```diff
@@ -657,11 +657,21 @@ void player::pause()
             }
         }
         effect &e = get_effect( effect_bleed, bp_id );
-        time_duration penalty = 1_turns * ( encumb( bodypart_id( "hand_r" ) ) + encumb(
-                                                bodypart_id( "hand_l" ) ) );
+        int total_hand_encumb = 0;
+        for( const bodypart_id &part : get_all_body_parts_of_type( body_part_type::type::hand ) ) {
+            total_hand_encumb += encumb( part );
+        }
+        time_duration penalty = 1_turns * total_hand_encumb;
```

Comment:
I feel code like this also needs some revisitation when more than two hands/arms/legs are possible. Take bleeding stopping here as an example, are 2 usable hands out of 10 the same as 2 usable hands out of 2? I'd argue they are more or less the same, but the current code makes it so that your 8 very encumbered hands will penalize your 2 usable hands.

Reply:
yes i agree. i've got some general plans for it i discussed in the discord, but i feel that this pr needs to land first because it also has some genericization code to help out with this infrastructure.